### PR TITLE
Onboarding Status page

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
image

**Features**
- Onboarding Status page

### CODE REVIEW

File: src/pages/admin/onboarding.tsx [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+    const statistics = {
+      teacher: {
+        complete: teacherOnboardedCount!,
+        full: teacherOnboardedCount! + teacherNotOnboardedCount!,
+      },
+      student: {
+        complete: studentOnboardedCount!,
+        full: studentOnboardedCount! + studentNotOnboardedCount!,
+      },
+    };
```
Comment (SiravitPhokeed) [inline src/pages/admin/onboarding.tsx:356]: Feels like this should be in `backend`.

File: src/utils/backend/person/teacher.ts [hunk 2]
```diff
@@ -17,11 +17,15 @@ import {
...
   DatabaseClient,
+  LangCode,
 } from "@utils/types/common";
 import { ImportedTeacherData, Teacher } from "@utils/types/person";
 
 // Miscellaneous
 import { prefixMap, subjectGroupMap } from "@utils/maps";
+import { IndividualOnboardingStatus } from "@utils/types/admin";
+import { nameJoiner } from "@utils/helpers/name";
+import { Database } from "@utils/types/supabase";
```
Comment (SiravitPhokeed) [inline src/utils/backend/person/teacher.ts:28]: Have I mentioned that imports should be grouped?

File: src/utils/backend/person/teacher.ts [hunk 3]
```diff
@@ -264,3 +268,62 @@ export async function getClassAdvisorAt(
 
   return { data: data!, error: null };
 }
+
+export async function getOnBoardTeacherData(
```
Comment (SiravitPhokeed) [inline src/utils/backend/person/teacher.ts:272]: “Onboard” is one word, so a better name would be `getTeacherOnboardStatus`, or something like that.

**Review by SiravitPhokeed — COMMENTED** [review]
_LGTM!_

**Review by Jimmy-Tempest — DISMISSED** [review]
LGTM!

**Review by Jimmy-Tempest — APPROVED** [review]

**Pull request merged**